server/gitrest/packages/gitrest-base/src/routes/summaries.ts

```diff
@@ -127,6 +128,7 @@ async function createSummary(
         repoManager,
         lumberjackProperties,
         externalWriterConfig?.enabled ?? false,
+        { enableLowIoWrite },
```

Comment:
Just for learning, how did the `GitWholeSummaryManager` consumed passed in parameter `enableLowIoWrite` ? Didn't find any other changes using this parameter.

Reply:
Hey Xin, GitHub doesn't expand the Diff for the `GitWholeSummaryManager` change because it's over 600 lines changed.
![image](https://user-images.githubusercontent.com/14362200/207119708-d07d3a44-9f86-46ee-9d18-7639d647a35c.png)
You should be able to see the changes with this param (and the actual majority of this PR) by expanding that file diff